charts/tempo/README.md

```diff
@@ -13,7 +13,7 @@ Grafana Tempo Single Binary Mode
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
 | affinity | object | `{}` |  |
-| config | string | `"multitenancy_enabled: {{ .Values.tempo.multitenancyEnabled }}\nsearch_enabled: {{ .Values.tempo.searchEnabled }}\ncompactor:\n  compaction:\n    compacted_block_retention: {{ .Values.tempo.retention }}\ndistributor:\n  receivers:\n    {{- toYaml .Values.tempo.receivers | nindent 8 }}\ningester:\n  {{- toYaml .Values.tempo.ingester | nindent 6 }}\nserver:\n  {{- toYaml .Values.tempo.server | nindent 6 }}\nstorage:\n  {{- toYaml .Values.tempo.storage | nindent 6 }}\noverrides:\n  {{- toYaml .Values.tempo.global_overrides | nindent 6 }}\n"` | Tempo configuration file contents |
+| config | string | `"multitenancy_enabled: {{ .Values.tempo.multitenancyEnabled }}\nsearch_enabled: {{ .Values.tempo.searchEnabled }}\nmetrics_generator_enabled: {{ .Values.tempo.metricsGenerator.enabled }}\ncompactor:\n  compaction:\n    compacted_block_retention: {{ .Values.tempo.retention }}\ndistributor:\n  receivers:\n    {{- toYaml .Values.tempo.receivers | nindent 8 }}\ningester:\n  {{- toYaml .Values.tempo.ingester | nindent 6 }}\nserver:\n  {{- toYaml .Values.tempo.server | nindent 6 }}\nstorage:\n  {{- toYaml .Values.tempo.storage | nindent 6 }}\noverrides:\n  {{- toYaml .Values.tempo.global_overrides | nindent 6 }}\n  {{- if .Values.tempo.metricsGenerator.enabled }}\n      metrics_generator_processors: \n      - 'service-graphs'\n      - 'span-metrics'\nmetrics_generator:\n      storage:\n        path: \"/tmp/tempo\"\n        remote_write:\n          - url: {{ .Values.tempo.metricsGenerator.remoteWriteUrl }}\n  {{- end }}\n"` | Tempo configuration file contents |
```

Comment:
This needs fixing. Results in:
![Screenshot 2022-07-18 at 15 42 23](https://user-images.githubusercontent.com/50638541/179525087-8093e538-d42c-4ee9-a929-c6d394437c26.png)

Reply:
I think it's a feature 😄 

It's the same on the documentation that is currently in main: https://github.com/grafana/helm-charts/tree/main/charts/tempo#values